packages/plugins/contracts/test/script/Cases.s.sol

```diff
@@ -40,62 +37,15 @@ contract Simple is Script, Sphinx {
         fallbackContract.set(1);
 
         // Deploys contract that deploys another contract in its constructor using create2
-        // The contract deployed is labeled
         ConstructorDeploysContract constructorDeploysContract = new ConstructorDeploysContract{
             salt: bytes32(uint(1))
         }(1);
-        sphinxLabel(
-            address(constructorDeploysContract.myContract()),
-            "contracts/test/ConstructorDeploysContract.sol:DeployedInConstructor"
-        );
 
         // Deploys contract that deploys another contract in its constructor using create3
-        // Both the parent and child are labeled
         bytes memory constructorDeploysContractInitCode = abi.encodePacked(
             type(ConstructorDeploysContract).creationCode,
             abi.encode(2)
         );
```

Comment:
We still deploy this exact contract. (It's below all of the deleted code). It's worth keeping this contract because we use it to test that we assign the child contract to the correct contract. Also, we use it to test that the `CREATE3` proxy is in the `unlabeledAddresses` array in the `ParsedConfig`